tests/conftest.py

```diff
@@ -1,11 +1,31 @@
 import asyncio
+import uuid
+from distutils.version import StrictVersion
+from os import environ as ENV
 
 import pytest
 
 from aiodocker.docker import Docker
 from aiodocker.exceptions import DockerError
 
 
+_api_versions = {
```

Comment:
For us, because at that point you are testing the good implementation of Docker of a backward compatibility API.

Let's say Docker 17.06 has backward compatibility for the API 1.27. We test the API 1.27 already with Docker 17.03. If we now test the 1.27 API on Docker 17.06, the only thing we test is that Docker has well implemented their backward compatibility API. It's not our job.